doc/source/yt4differences.rst

```diff
@@ -159,24 +157,20 @@ It should be noted that the
     # generate an octree
     octree = ds.octree(left, right, n_ref=64)
 
+    ds.sph_smoothing_style = "scatter"
+
     # the density will be calculated using SPH scatter
     density = octree[('PartType0', 'density')]
 
     # this will return the x positions of the octs
     x = octree[('index', 'x')]
 
 The above code can be modified to use the scatter approach by using
-``ds.sph_smoothing_style = 'gather'`` before any field access. The octree also
-accepts ``over_refine_factor`` which works just like the ``over_refine_factor``
-parameter in yt-3.0 that could be passed to ``yt.load``, and determines how many
-particles are in each leaf.
-
-The ``density_factor`` keyword allows the construction of dense octrees
-trees. In a traditional octree, if a leaf has more particles that a critical
-value `n_ref`, then it divides into 8 new children (hence the name oct). The
-value of `density_factor` allows the node to divide into 2^(3*density_factor)
-zones instead. This creates an octree structure similar that used by AMR codes
-like FLASH that make use of an octree of grid patches.
+``ds.sph_smoothing_style = 'gather'`` before any field access.
```

Comment:
so sph_smoothing_style is accessible from both the ds and the octree objects? Do we want to document that here too? e.g what happens if the sph_smoothing_style is set on the ds object but not the octree? What happens when one is changed?

Reply:
I didn't even catch this. I think that is a bad precedent to set. I have instead reverted to just defining those properties on the `ds` and then I the octree grabs them each time it smooths a field so they can be updated/viewed throughout a script